cli/azd/test/functional/testdata/samples/springapp/azure.yaml

```diff
@@ -0,0 +1,13 @@
+name: webapp
+metadata: 
+  template: azd-test/webapptest@v1
+services:
+  sweb:
+    project: ./
+    host: springapp
+    language: java
+    spring:
+      deploymentName: default
+infra:
+  provider: terraform
```

Comment:
NIT: I would prefer if we wrote this small sample with bicep instead of terraform, I don't think it would be that hard to port this to bicep.

If it's a big ask to move it over to Bicep, happy to land this as is, since it's easy enough for me to port it to Bicep based on the template, but I think it would help the team out a lot if these test applications were mostly in Bicep.